.github/workflows/release-brew.yaml

```diff
@@ -174,14 +168,12 @@ jobs:
     - name: Fetch secrets
       run: |
         echo "BOT_EMAIL=$(aws secretsmanager get-secret-value --secret-id BOT_EMAIL | jq -r '.SecretString')" >> $GITHUB_ENV
-        echo "HOMEBREW_GITHUB_API_TOKEN=$(aws secretsmanager get-secret-value --secret-id RELEASE_CI_ACCESS_TOKEN | jq -r '.SecretString')" >> $GITHUB_ENV
-        echo "FORK_REPO=https://$HOMEBREW_GITHUB_API_TOKEN@github.com/$BOT_USER/homebrew-$(echo $TAP |cut -d / -f 2).git" >> $GITHUB_ENV
-        echo "GITHUB_TOKEN=$HOMEBREW_GITHUB_API_TOKEN" >> $GITHUB_ENV
+        echo "FORK_REPO=https://$(aws secretsmanager get-secret-value --secret-id RELEASE_CI_ACCESS_TOKEN | jq -r '.SecretString')@github.com/$BOT_USER/homebrew-$(echo $TAP |cut -d / -f 2).git" >> $GITHUB_ENV
 
     - name: Configure git user name and email
       run: |
         git config --global user.name ${{ env.BOT_USER }}
-        git config --global user.email BOT_EMAIL
+        git config --global user.email $BOT_EMAIL
```

Comment:
Shouldn't this be `${{ env.BOT_EMAIL }}`

Reply:
I believe those can be used interchangeably, but arguably we should pick a consistent syntax. Will fix.

Reply:
Done.